src/main/scala/com/github/whaitukay/utils/UtilWrapper.scala

```diff
@@ -16,6 +16,18 @@ object UtilWrapper {
     FileUtils.listFiles(filepath).asJava
   }
 
+  def deleteFileOrDir(filepath: String): AnyVal = {
+    FileUtils.deleteFileOrDir(filepath)
+  }
+
+  def rename(srcPath:String, dstPath: String): Boolean = {
+    FileUtils.rename(srcPath, dstPath)
+  }
+
+  def copyMoveDir(srcPath:String, dstPath: String, deleteSrc: Boolean): Boolean = {
+    FileUtils.copyMoveDir(srcPath, dstPath, deleteSrc)
+  }
+
```

Comment:
Since these all operate on files and directories in the same manner, maybe we should omit references to File/Dir in their names?
`deleteFileOrDir` -> `delete`
`copyMoveDir` -> `copyMove`